tsMuxer/muxerManager.h

```diff
@@ -45,7 +45,7 @@ class MuxerManager
 
     void parseMuxOpt(const std::string& opts);
     int getTrackCnt() { return (int) m_metaDemuxer.getCodecInfo().size();}
-
+    bool getHevcFound() { return (bool) m_metaDemuxer.m_HevcFound; }

```

Comment:
`m_HevcFound` is already a `bool`. The cast is redundant. Also, this should be a `const` function.

Reply:
Hi xavery, cast (bool) removed. I don't think it should be a const, the value of m_HevcFound varies with each line of the meta file.